doc/nrf/installation/install_ncs.rst

```diff
@@ -0,0 +1,616 @@
+This includes everything that is required by Zephyr's :ref:`zephyr:getting_started` together with additional tools and Python dependencies that the |NCS| uses.
+
+.. rst-class:: numbered-step
+
+Update operating system
+***********************
+
+Before you start setting up the toolchain, install available updates for your operating system.
+See :ref:`requirements` for information on the supported operating systems.
+
+.. _installing_vsc:
+
+.. rst-class:: numbered-step
+
+Install prerequisites
+*********************
+
+Depending on your preferred development environment, install the following tools:
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      * The latest version of |VSC| for your operating system from the `Visual Studio Code download page`_.
+      * In |VSC|, the latest version of the `nRF Connect for VS Code Extension Pack`_.
+      * The latest version of the :ref:`requirements_clt` package.
+        Download it from the `nRF Command Line Tools`_ page.
+
+   .. group-tab:: Command line
+
+      * The latest version of `nRF Util development tool`_, a unified command line utility for Nordic products.
+
+        .. note::
+            After downloading the nRF Util executable, move it to a directory that is in the system :envvar:`PATH`.
+            On macOS and Linux, the downloaded file also needs to be given execute permission by typing `chmod +x nrfutil` or by checking the checkbox in the file properties.
+
+      * The latest version of the :ref:`requirements_clt` package.
+        Download it from the `nRF Command Line Tools`_ page.
+
+        .. note::
+            After downloading and installing the tools, add nrfjprog to the system :envvar:`PATH` in the environment variables.
+
+.. _gs_installing_toolchain:
+.. _gs_installing_tools:
+
+.. rst-class:: numbered-step
+
+Install the |NCS| toolchain
+***************************
+
+The |NCS| :term:`toolchain` includes the Zephyr SDK and then adds on top of it tools and modules required to build |NCS| samples and applications.
+These include the :ref:`required tools <requirements_toolchain_tools>`, the Python dependencies, and the :ref:`GN tool <ug_matter_gs_tools_gn>` for creating :ref:`ug_matter` applications.
+
+When you first install the |NCS|, it is recommended to install the latest released versions of the SDK and the toolchain.
+
+Depending on your preferred development environment, complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for Visual Studio Code
+
+      1. Open the nRF Connect extension in |VSC| by clicking its icon in the :guilabel:`Activity Bar`.
+      #. In the extension's :guilabel:`Welcome View`, click on :guilabel:`Manage toolchains`.
+         The list of actions appears in the |VSC|'s quick pick.
+      #. Click :guilabel:`Install Toolchain`.
+         The list of available toolchain versions appears in the |VSC|'s quick pick.
+      #. Select the toolchain version to install.
```

Comment:
```suggestion
      #. In the Toolchain Management menu, click :guilabel:`Install Toolchain`.
      #. Select the toolchain version to install.
```

These "The list of ABCs appears in the Visual Studio Code's quick pick" sentences look a bit broken, and they're a bit redundant. I suggest we remove them.